schema/game_author.rb

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS 'authors' (
+  'id' INTEGER PRIMARY KEY AUTOINCREMENT,
+  'first_name' TEXT NOT NULL,
+  'last_name' TEXT NOT NULL,
+  'publish_date' DATE NOT NULL,
+  'archived' BOOLEAN NOT NULL DEFAULT FALSE,
+);
+
+CREATE TABLE IF NOT EXISTS 'game' (
+  'id' INTEGER PRIMARY KEY AUTOINCREMENT,
+  'multiplayer' TEXT NOT NULL,
+  'last_played_at' DATE NOT NULL,
+  'archived' BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
'archived' is also defined in the item class. So no need to add it here. Kindly add a foreign key referencing item_id to establish the relationship